README.md

```diff
@@ -33,6 +33,7 @@ The following potions are present in this module:
 * Super Heal - Restores a vast amount of HP
 * Sacred - Regenerates 3 HP per second for 10 seconds, increases the user's base max health by 25% for 30 seconds, and also cures poison status.
 * Ultra Heal - Restores a very vast amount of HP for 2 seconds.
+* Divergence - Coincidentally heals and damages the player for 10 seconds, but heals in larger amounts.
```

Comment:
Could you also add an entry to the bottom of this file indicating that you made a variant of the base bottle for your new potion graphic? Also may need to update the description if you switch it over to HARM + REGEN (my recommendation)